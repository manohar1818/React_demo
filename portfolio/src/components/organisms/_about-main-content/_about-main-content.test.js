import React from 'react';
import { create } from "react-test-renderer";
import { render ,fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from './_about-main-content';

describe("About component",() =>{
    test("Matches the snapshot",() => {
        const about = create(<About />);
        expect(about.toJSON()).toMatchSnapshot();

    })
    test("about element is rendered",() => {
        const { getByTestId, getByText } = render(<About />);
        expect(getByTestId("aboutMainContent")).toBeInTheDocument();
        
    });
   
   

});