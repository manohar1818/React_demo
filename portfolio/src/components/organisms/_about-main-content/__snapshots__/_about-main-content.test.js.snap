// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component Matches the snapshot 1`] = `
<div
  className="makeStyles-aboutMainContent-1"
  data-testid="aboutMainContent"
>
  <h6
    className="MuiTypography-root makeStyles-about-2 MuiTypography-subtitle2"
  >
     I am Ruchitha from Keshav Memorial Institute of Technology.I'm passionate about coding and learning new technologies.In free time I like to chill at home watching movies and colouring .I love to be surrounded by positive and motivating people. 
  </h6>
  <div
    className="makeStyles-list-4"
  >
    <div
      className="makeStyles-list-6"
      data-testid="aboutCard"
    >
      <h6
        className="MuiTypography-root makeStyles-heading-7 MuiTypography-subtitle1"
      >
        Technical Skills
      </h6>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Java
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        DBMS
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        React
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Spring
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Spring boot
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        REST
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Networking
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Linux
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        HTML,CSS
      </p>
    </div>
    <div
      className="makeStyles-list-6"
      data-testid="aboutCard"
    >
      <h6
        className="MuiTypography-root makeStyles-heading-7 MuiTypography-subtitle1"
      >
        Projects
      </h6>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Detection and Prediction of Clouds
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Gate Pass Management System
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Making Data Based Decisions Using Google Analytics
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Online Doctor Appointment
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Spring boot REST API which gives information about movies and associated actors
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Spring boot application which gives information about courses and related topics
      </p>
    </div>
  </div>
</div>
`;
